core/os/dir_access.cpp

```diff
@@ -250,6 +250,32 @@ DirAccess *DirAccess::create_for_path(const String &p_path) {
 	return da;
 }
 
+Vector<String> DirAccess::list_files(const String &p_path, const String &p_file_ext) {
```

Comment:
I think adding a method to list contents of a folder easily (like `ls` or `dir`) makes sense, but I'd probably use a somewhat different API to be able to list files, directories or both, and allow including hidden files, etc.

For this PR I think it would make sense to keep this local, and just implement `list_gdap_plugins` in `platform/android/export/export.cpp`. Implementing a more generic API in `DirAccess` should be in its own PR and would likely require more design discussion.